src/Integration/Persistence/SolutionBindingSerializer.cs

```diff
@@ -98,14 +94,17 @@ private void AddSolutionItemFile(string configFile)
         {
             Debug.Assert(!string.IsNullOrWhiteSpace(configFile), "Invalid configuration file");
 
-            Project solutionItemsProject = this.projectSystemHelper.GetSolutionItemsProject();
+            var projectSystemHelper = this.serviceProvider.GetService<IProjectSystemHelper>();
```

Comment:
We only need the projectSystemHelper at this point, so I moved the service request here.
Other tests that only need to read will need to register another somewhat unrelated service and we'll wonder why next time we will look at that code.